webapp/src/routes/home/game/index.js

```diff
@@ -33,48 +40,37 @@ export class Game {
   }
 
   showLauncher() {
-    this.launcher = new Launcher({
-      loginViaGoogle: this.loginViaGoogle,
-      containerSize: {
-        width: this.width,
-        height: this.height,
-      },
-    });
-
     this.stage.addChild(this.launcher.stage);
   }
 
   showGame() {
     this.background = new Background({ width: this.width, height: this.height });
     this.warehouse = new Warehouse({ rendererWidth: this.width });
     this.fenceGroup = new FenceGroup({ rendererWidth: this.width, rendererHeight: this.height });
-    this.userInterface = new UserInterface({ rendererWidth: this.width });
-    this.chicken = new Animal({
-      rendererWidth: this.width,
-      rendererHeight: this.height,
-      type: ANIMALS.chicken,
-      positionNumber: 1,
-      amount: 10,
-      level: 10,
-    });
-    this.chick = new Animal({
+    this.userInterface = new UserInterface({ rendererWidth: this.width, game: this.game });
+
+    const animals = this.game.fields.map(({ type, position, amount, level }) => new Animal({
```

Comment:
`this.game.fields.map(({ position: positionNumber, ...other }) => new Animal({ positionNumber, ...other }); `

Reply:
True, I know it will work like that.